run.sh

```diff
@@ -16,14 +16,20 @@ while [ -n "$1" ]; do
 done
 
 SDC_HOME=${SDC_HOME:-$(mktemp -d)}
+DB_HOME=$SDC_HOME/data
+LOGS_HOME=$SDC_HOME/logs
+
+export SDC_HOME DB_HOME LOGS_HOME
+
+mkdir -p $DB_HOME
+mkdir -p $LOGS_HOME
+chmod 0700 $SDC_HOME
+chmod 0700 $DB_HOME
+chmod 0700 $LOGS_HOME
```

Comment:
again isn't this also done in the app itself?

Reply:
Because we want to have the db file as `~/.securedrop_client/data`, if we have it under `~/.securedrop_client/` then I can remove these lines. As the database must come up (using Alembic) before the application starts.

Reply:
true, we gotta create the database before starting the application - but see my comment [here](https://github.com/freedomofpress/securedrop-client/pull/86#discussion_r229545079), we can at least delegate these folder creation to the application (the less logic we have duplicated the better)